examples/basic-typescript/app/[[...path]]/page.tsx

```diff
@@ -0,0 +1,27 @@
+import { notFound } from 'next/navigation'
+
+import { Page as MakeswiftPage } from '@makeswift/runtime/next'
+import { getSiteVersion } from '@makeswift/runtime/next/server'
+
+import { client } from '@/lib/makeswift/client'
+
+type ParsedUrlQuery = { path?: string[] }
+
+export async function generateStaticParams() {
+  const pages = await client.getPages()
+
+  return pages.flatMap(page => ({
```

Comment:
Why are we using `flatMap` here?

Reply:
Good question. I actually used the preview of the docs to do this migration. This was a c/p from there. surprised it worked. Glad you caught this 🙌🏼 

Fixing the docs